omegaconf/grammar/OmegaConfGrammarParser.g4

```diff
@@ -68,15 +75,14 @@ primitive:
         | interpolation
       )+;
 
-// Same as `primitive` except that `COLON` and interpolations are not allowed.
+// Same as `primitive` except that `COLON`, interpolations and quoted values are not allowed.
```

Comment:
looks like the comment was outdated (claimed interpolation was already not allowed).
I remember that we agreed on it at some point, so I am not sure how they crept back.

Reply:
Back when interpolations were allowed, there was an `| interpolation` added to the list (which isn't there anymore in the current master, so the comment is correct in that sense).
However, allowing `QUOTED_VALUE` indeed made it possible to use quoted strings that may contain interpolations.